.circleci/config.yml

```diff
@@ -98,27 +132,26 @@ workflows:
               ignore: /.*/
             tags:
               only: /^v[0-9].[0-9].[0-9]+.*/
-      - publish-dev:
+      - spawn-stability-tests-job:
           requires:
+            - lint
+            - test
             - build
           filters:
             branches:
-              only: master
+              only: /master/
             tags:
               ignore: /.*/
 
 jobs:
-  setup-and-lint:
+  setup:
     executor: golang
     steps:
       - checkout
       - restore_module_cache
       - persist_to_workspace:
           root: ~/
           paths: project
-      - run:
-          name: Install bazzar
-          command: sudo apt update && sudo apt-get install bzr -y
```

Comment:
We don't need this as Go now fetches dependencies from the Go modules proxy server instead of using VCS systems directly.